src/jshint.js

```diff
@@ -409,15 +409,13 @@ var JSHINT = (function() {
   }
 
   // Tracking of "internal" scripts, like eval containing a static string
-  function addInternalSrc(elem, src) {
-    var i;
-    i = {
+  function addInternalSrc(elem, token) {
```

Comment:
Since you're refactoring, it makes sense to rename this more appropriate, perhaps: `addEvalCode`. https://tc39.github.io/ecma262/#sec-types-of-source-code